recipe.yaml

```diff
@@ -52,11 +55,26 @@
       path: /home
       mode: 0777
       recurse: yes
+
   - copy:
       src: ./files/passwd
       dest: '{{ home_path }}/passwd'
       mode: 0777
 
+  - name: Create /usr/share/packit directory
+    file:
+      state: directory
+      path: /usr/share/packit
+  - name: Copy packit.wsgi file
+    copy:
+      src: files/packit.wsgi
+      dest: /usr/share/packit/packit.wsgi
+  - name: Change permissions packit.wsgi
```

Comment:
This failed for my unknown reason. It seems, like after httpd installation apache user does not exist.
~~~
TASK [Copy packit.wsgi file] *******************************************************************************************************
changed: [docker-io-usercont-packit-service-20190412-124913102420-cont]

TASK [Change permissions packit.wsgi] **********************************************************************************************
 [WARNING]: Module invocation had junk after the JSON data: error while running runtime: exit status 1
fatal: [docker-io-usercont-packit-service-20190412-124913102420-cont]: FAILED! => {"changed": false, "gid": 0, "group": "root", "mode": "0644", "msg": "chown failed: failed to look up user apache", "owner": "0", "path": "/usr/share/packit/packit.wsgi", "size": 80, "state": "file", "uid": 0}

PLAY RECAP *************************************************************************************************************************
docker-io-usercont-packit-service-20190412-124913102420-cont : ok=7    changed=6    unreachable=0    failed=1   

time="2019-04-12T12:52:07+02:00" level=error msg="exit status 2" 
time="2019-04-12T12:52:07+02:00" level=error msg="exit status 2" 
Getting image source signatures
Copying blob sha256:b7ba3be6a0d6426356fc49f538f0f929ed00e51a0a90906e354a345d46e23a8b
Copying blob sha256:76476a644c26ea87234435734b3fbe7b2a02ae3223a2772966deee1ed5b37631

~~~

@jpopelka @TomasTomecek Can you help me what's wrong?

Reply:
Let me see.

Reply:
I suspect this would be a limitation of rootless buildah. Will try to build as root.

Reply:
In the end, the problem was the NSS_WRAPPER_PASSWD nonsense we have to do for sake of openshift

I have added entries for root and apache in our custom passwd file.

Reply:
Sorry;) I did git push force and did not include your changes.